cmd/handlers/handlers.go

```diff
@@ -26,15 +26,27 @@ func CreateURL(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	id := utils.GenerateID()
-	url := storage.URL{ID: id, Original: string(b), Short: fmt.Sprintf("%s/%s", config.AppConfig.BaseURL, id)}
+	userID := r.Context().Value(utils.ContextKey("userID")).(string)
+	url, e := repository.MakeURL(string(b), userID)
```

Comment:
Лучше именовать ошибки везде `err`
```suggestion
	url, err := repository.MakeURL(string(b), userID)
```